Makefile.include

```diff
@@ -1,8 +1,15 @@
 INSTALL=/usr/bin/install
 OBS_BACKEND_PREFIX=/usr/lib/obs/server
+OBS_BACKEND_DATA_DIR=/srv/obs
+OBS_BACKEND_LOG_DIR=$(OBS_BACKEND_DATA_DIR)/log
+OBS_BACKEND_SERVICE_LOG_DIR=$(OBS_BACKEND_DATA_DIR)/service/log
 OBS_DOCUMENT_ROOT=/srv/www/obs
 OBS_API_PREFIX=$(OBS_DOCUMENT_ROOT)/api
+OBS_API_LOG_DIR=$(OBS_API_PREFIX)/log
 OBS_APIDOCS_PREFIX=$(OBS_DOCUMENT_ROOT)/docs
+OBS_SRCSERVER_PORT=5352
+OBS_REPOSERVER_PORT=5252
+OBS_SERVICESERVER_PORT=5152
```

Comment:
So far I see no reason to make ports configurable.

Reply:
Ports should be configurable because they may conflict with other services.
For example, Fedora uses port 5252 for `milter-greylist`.